proposals/market_perp_btcusd.go

```diff
@@ -131,40 +131,45 @@ func NewBTCUSDPerpetualMarketProposal(
 							"base:BTC",
 							"quote:USD",
 							"class:fx/crypto",
-							"monthly",
+							"perpetual",
 							"sector:crypto",
 						}, extraMetadata...),
 						PriceMonitoringParameters: &vega.PriceMonitoringParameters{
 							Triggers: []*vega.PriceMonitoringTrigger{
 								{
 									Horizon:          43200,
-									Probability:      "0.9999999",
-									AuctionExtension: 600,
+									Probability:      "0.9999",
+									AuctionExtension: 300,
+								},
+								{
+									Horizon:          14400,
+									Probability:      "0.9999",
+									AuctionExtension: 180,
 								},
 								{
-									Horizon:          300,
+									Horizon:          3600,
 									Probability:      "0.9999",
-									AuctionExtension: 60,
+									AuctionExtension: 120,
 								},
 							},
 						},
-						LpPriceRange: "0.5",
+						LpPriceRange: "0.05",
 						LiquidityMonitoringParameters: &vega.LiquidityMonitoringParameters{
 							TargetStakeParameters: &vega.TargetStakeParameters{
 								TimeWindow:    3600,
 								ScalingFactor: 10,
 							},
-							TriggeringRatio:  "0.0",
+							TriggeringRatio:  "0.7",
 							AuctionExtension: 1,
 						},
 						RiskParameters: &vega.NewMarketConfiguration_LogNormal{
 							LogNormal: &vega.LogNormalRiskModel{
-								RiskAversionParameter: 0.0001,
-								Tau:                   0.0000190129,
+								RiskAversionParameter: 0.000001,
+								Tau:                   0.0001140771161,
```

Comment:
I'd be up for a more funky `tau`; let's aim for like a 2 minute tau so we get proper degen level leverage.

Reply:
Ok, went for 2mins and used 10x mainnet linear slippage factor